api_app/api/routes/airlock.py

```diff
@@ -68,7 +68,7 @@ async def get_all_airlock_requests_by_workspace(
     return AirlockRequestWithAllowedUserActionsInList(airlockRequests=airlock_requests_with_allowed_user_actions)
 
 
-@airlock_workspace_router.get("/workspaces/{workspace_id}/requests/{airlock_request_id}", status_code=status.HTTP_200_OK, response_model=AirlockRequestInResponse, name=strings.API_GET_AIRLOCK_REQUEST, dependencies=[Depends(get_current_workspace_owner_or_researcher_user), Depends(get_workspace_by_id_from_path)])
```

Comment:
@jjgriff93  this would be the endpoint that the Airlock Manager uses to review Airlock requests? Feels like a pretty major bug, wondering why we didn't run into this, although maybe we just were always Workspace Owners in our testing environments :/

Reply:
Yes I think it's because we just added the airlock manager role to our main Microsoft identities, so we would have both been workspace managers